packages/jest-openapi/src/openapi-validator/lib/classes/RequestPromiseResponse.js

```diff
@@ -4,7 +4,7 @@ class RequestPromiseResponse extends AbstractResponse {
   constructor(res) {
     super(res);
     this.status = res.statusCode;
-    this.body = res.body.replace(/"/g, '');
+    this.body = res.request.headers.accept === 'application/json' ? res.body : res.body.replace(/"/g, '');
```

Comment:
Thanks for the update to `chai-openapi-response-validator`. We should probably keep `jest-openapi` consistent by copying the update to here